include/zck.h.in

```diff
@@ -58,125 +58,148 @@ typedef struct zckDL zckDL;
 
 typedef size_t (*zck_wcb)(void *ptr, size_t l, size_t c, void *dl_v);
 
+#ifdef _WIN32
+    #define ZCK_WARN_UNUSED
+    typedef ptrdiff_t ssize_t;
+    #ifdef ZCHUNK_STATIC_LIB
+        #define ZCK_PUBLIC_API
+    #else
+        #ifdef ZCHUNK_EXPORTS
+            #define ZCK_PUBLIC_API __declspec(dllexport)
+        #else
+            #define ZCK_PUBLIC_API __declspec(dllimport)
+        #endif
+    #endif
+#else
+    #define ZCK_WARN_UNUSED __attribute__ ((warn_unused_result))
+    #define ZCK_PUBLIC_API __attribute__((visibility("default")))
+#endif
+
+#ifdef _WIN32
```

Comment:
ZCK_LOG_ERROR is already defined as an ENUM.  Within the library, `zck_log()` should be used for logging.  For the utilities, this should be defined somewhere other than `zck.h.in` since it's only used by the utilities and should be given a new name (maybe something simple like `LOG_ERROR`).

Reply:
I've defined `LOG_ERROR` in `src/util_common.h` and `test/util.h` now. It's a bit annoying to have the same definition twice, but idk how to solve that. We could make another shared header-only file.

Please let me know what you think!